joomla/joomla-cms
# [4.0] Namespace  com_users

Pull Request for Issue # .

### Summary of Changes
This simple PR converts com_users component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_users .zip )

2. Frontend Testing: Register for an account, try forgot username, forgot password, edit profile, make sure it is working as expected

3. Backend testing: Try add groups, access levels, user...., make make sure it is working as expected
